### PR TITLE
Create Agreement Number

### PR DESCRIPTION
**Pull request by fareed-mohammed**
https://eaflood.atlassian.net/browse/SFI-675

### CODE REVIEW

**Review by steve-dickinson — COMMENTED** [review]
There is a generateAgreementNumber function in the agreement-number module

File: app/routes/v2/standards/index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = [{
       const response = await getPollingResponse(request.yar.id, '/standards')
       if (response) {
         console.info('Standards request received', response)
-        await cache.update('apply-journey', request.yar.id, { agreementNumber: response.agreementNumber, standards: response.standards })
+        await cache.update('apply-journey', request.yar.id, { agreementNumber: response.agreementNumber ?? `AG${new Date().getTime()}`, standards: response.standards })
```
Comment (steve-dickinson) [inline app/routes/v2/standards/index.js:17]: There is a generateAgreementNumber function in the agreement-number module

**Review by steve-dickinson — COMMENTED** [review]
LGTM

File: app/routes/v2/standards/index.js (hunk 1 above)
Comment (fareed-mohammed) [reply app/routes/v2/standards/index.js]: agree

**Review by steve-dickinson — APPROVED** [review]
LGTM

**Pull request merged**